graftm/decorator.py

```diff
@@ -1,10 +1,39 @@
-
+#!/usr/bin/env python
+###############################################################################
+#                                                                             #
+# This program is free software: you can redistribute it and/or modify        #
+# it under the terms of the GNU General Public License as published by        #
+# the Free Software Foundation, either version 3 of the License, or           #
+# (at your option) any later version.                                         #
+#                                                                             #
+# This program is distributed in the hope that it will be useful,             #
+# but WITHOUT ANY WARRANTY; without even the implied warranty of              #
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the                #
+# GNU General Public License for more details.                                #
+#                                                                             #
+# You should have received a copy of the GNU General Public License           #
+# along with this program. If not, see <http://www.gnu.org/licenses/>.        #
+#                                                                             #
+###############################################################################
+__author__ = "Joel Boyd, Ben Woodcroft"
```

Comment:
Hey I think including the copyright notice is a good idea, but I think the extra `__author__` is a bit too much to maintain - WDYT?